ftl/core/adding.ftl

```diff
@@ -6,4 +6,6 @@ adding-history = History
 adding-note-deleted = (Note deleted)
 adding-shortcut = Shortcut: { $val }
 adding-the-first-field-is-empty = The first field is empty.
-adding-you-have-a-cloze-deletion-note = You have a cloze deletion note type but have not made any cloze deletions. Proceed?
+adding-missing-cloze = You have a cloze notetype but have not made any cloze deletions.
+adding-cloze-outside-cloze-notetype = Cloze deletion can only be used on cloze notetypes.
+adding-cloze-outside-cloze-field = Cloze deletion can only be used in fields which use the 'cloze:' filter (see 'Cards...').
```

Comment:
This is nice, it will hopefully cut down on users adding clozes to the Extra field accidentally. Since the strings in core/ are intended to be used across the different clients though, it would be better if the reference to the desktop "Cards..." could be made more generic, eg something like:

Cloze deletion can only be used in fields which use the 'cloze:' filter. This is typically the first field.

or

Cloze deletion can only be used in fields which use the 'cloze:' filter in the card template.

The former is probably more user-friendly.